src/main/java/com/undertheriver/sgsg/foler/domain/Folder.java

```diff
@@ -70,6 +70,10 @@ public void mapUser(User user) {
         this.user = user;
     }
 
+    public void delete() {
+        setDeleted(true);
```

Comment:
여기도 아래 PR과 같은
```suggestion
        delete()
```
객체에게 메시지주는형식이 좋을 것 같습니다.

Reply:
리뷰 감사합니다~!

`delete` 컬럼은 `BaseEntity`에 정의가 되어있고 각 엔티티들이 `BaseEntity`를 상속하고 있습니다.
그래서 `delete()`라는 메소드를 동시에 정의할 수가 없는 상황입니다.
그래서 어떻게 해결할까 고민하다가 `BaseEntity`의 `setDelete()` 함수를 `protected`로 지정해서 객체 내에서만 사용할 수 있도록 로직을 작성해봤습니다.

의견 부탁드립니다~!

Reply:
BaseEntity엔티티에서 메서드를 구현하면 될 것 같은데 제가 잘못알고있을까요?
```java
public void delete() {
  this.deleted = true;
}
```

Reply:
넵 맞습니다
그런데 지금 `User.delete()`의 경우에는 리턴 값이 `User`이고 `Memo.delete()`와 `Folder.delete()`는 리턴 값이 `void`입니다.
그래서 오버로딩이 안되더라구요

흠.. 생각해보니 부모 클래스인 `BaseEntity`에 `void` 타입을 리턴하는 `delete()` 정의해두고 `User.delete()` 는 오버라이딩을 하는게 낫겠군요. 코드도 줄어들고..

Reply:
넵 그럼 user의 delete방식을 바꿔야겠네요ㄷㄷㄷ 성찬님 User까지 코드수정부탁드립니다